docs/getting_started/discover_ui.md

```diff
@@ -4,22 +4,192 @@ description: Take a quick tour of the Back Office.
 
 # User interface
 
-## My dashboard
+## Dashboard
```

Comment:
Since this article belongs to a "Getting started" category and gives brief descriptions of various UI elements, I would strongly suggest leaving a short-ish section here, and creating a subisection in TOC called "Work with dashboard" that would contain two articles: Work with dashboard and Dashboard block reference.

Reply:
I think Dashboard needs a separate section with: Dashboard, Work with dashboard and dashboard blocks.